apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react';
+import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
+
+import type { PlacePickerRendererQueryResponse } from './__generated__/PlacePickerRendererQuery.graphql';
+import PlacePickerContent from './PlacePickerContent';
+
+type Props = {|
+  +type: string,
+  +defaultValue: ?string,
+|};
+
+export default class BookingDetail extends React.Component<Props> {
```

Comment:
I think the name shouldn't be `BookingDetail`

Reply:
🙈

Reply:
done